_includes/card-authorization-transaction.md

```diff
@@ -136,8 +136,7 @@ Content-Type: application/json
 | └➔&nbsp;`acquirerTransactionTime` | `string`  | `3DSECURE` or `SSL`. Indicates the transaction type of the acquirer.                                                                                                                                                                                                                                 |
 | └➔&nbsp;`id`                      | `string`  | {% include field-description-id.md resource="itemDescriptions" %}                                                                                                                                                                                                                                    |
 | └➔&nbsp;`nonPaymentToken`         | `string`  | Result of our own tokenization of the card used. Activated in POS on merchant or merchant group.                                                                                                                                                                                                     |
-| └➔&nbsp;`externalNonPaymentToken` | `string`  | Result of external tokenization. This value varies depending on cards, acquirer, customer, etc. For ICA cards, the token comes in response from Swedbank. For Mass Transit(SL) it is populated with PAR if it comes in response from the redeemer (Visa). If not, our own token (Mastercard / Amex). |
-|                                   |
+| └➔&nbsp;`externalNonPaymentToken` | `string`  | Result of external tokenization. This value varies depending on cards, acquirer, customer, etc. For ICA cards, the token comes in response from Swedbank. For Mass Transit(SL) it is populated with PAR if it comes in response from the redeemer (Visa). If not, our own token (Mastercard / Amex).                                                                                                                                                                 |                            
```

Comment:
Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)